pkg/virt-controller/watch/vmi_test.go

```diff
@@ -2476,6 +2476,89 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 				[]string{SuccessfulCreatePodReason}),
 		)
 
+		It("Should fail to get filesystem overhead if there are multiple CDI instances", func() {
+			cfg := cdiv1.CDIConfig{
```

Comment:
I understand that it works because we dont check what the informer returns, just the it returned 1 but cdiInformer should return CDI instance and not CDIConfig, so you should create cdi1 and cdi2 of cdiv1.CDI{}